src/Features/VisualBasic/Portable/Debugging/DataTipInfoGetter.vb

```diff
@@ -43,7 +43,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Debugging
 
                     Dim curr = expression
                     While True
-                        curr = curr.GetCorrespondingConditionalAccessExpression()
+                        curr = curr.GetParentConditionalAccessExpression()
```

Comment:
this method was renamed in VB to match teh C# equivalent.  It was a PITA trying to find each as NavigateTo would not work given teh different names.